cicd/pipelines/managing-nonversioned-and-versioned-cluster-tasks.adoc

```diff
@@ -6,13 +6,17 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-As a cluster administrator, installing the {pipelines-title} Operator creates variants of each default cluster task known as _versioned cluster tasks_ (VCT) and _non-versioned cluster tasks_ (NVCT). For example, installing the {pipelines-title} Operator v1.7 creates a `buildah-1-7-0` VCT and a `buildah` NVCT. 
+As a cluster administrator, installing the {pipelines-title} Operator creates variants of each default cluster task known as _versioned cluster tasks_ (VCT) and _non-versioned cluster tasks_ (NVCT). For example, installing the {pipelines-title} Operator v1.7 creates a `buildah-1-7-0` VCT and a `buildah` NVCT.
 
-Both NVCT and VCT have the same metadata, behavior, and specifications, including `params`, `workspaces`, and `steps`. However, they behave differently when you disable them or upgrade the Operator.  
+Both NVCT and VCT have the same metadata, behavior, and specifications, including `params`, `workspaces`, and `steps`. However, they behave differently when you disable them or upgrade the Operator.
+
+[IMPORTANT]
+====
```

Comment:
The supplementary style guide also has this note:

When citing deprecation timelines, refer to specific releases, such as the next release, only if that timeline is known to be accurate. Otherwise, use the phrase a future release because it accounts for the possibility of changes to the planned deprecation timeline.

Regarding snippets, many build/presentation systems really don't like them, I'd rather not take this risk for a deprecation note.

Reply:
To clarify, I mean here: "In <product_name> **<release>**, <name_of_capability_or_feature> "